html/answers/alps-q4.html

```diff
@@ -0,0 +1,139 @@
+<html>
+    <head>
+
+        <!-- Global site tag (gtag.js) - Google Analytics -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157837769-1"></script>
+        <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            HTML | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What is semantic HTML?">
+    </head>
+    <body>
+		
+		<img src="../../assets/CN_logo.png" height="72px" alt="Coding Ninjas Logo" />
+
+        <div>
+            <p>
+                <small>
+
+                    <a href="../../index.html">Home</a>
+                    &gt; <a href="../index.html">HTML Questions</a>
+                    &gt; What is semantic HTML?
+
+                </small>
+               
+            </p>
+        </div>
+
+
+        <content>
+            <div>
+                <h4>
+                    <h4><html>
+    <head>
+
+        <!-- Global site tag (gtag.js) - Google Analytics -->
+        <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157837769-1"></script>
+        <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'UA-157837769-1');
+        </script>
+        
+        <title>
+            HTML | InterQues | Coding Ninjas
+        </title>
+
+        <link rel="shortcut icon" type="image/png" href="../../assets/favicon.png"/>
+
+        <meta name="description" content="What is an image map?">
+    </head>
+    <body>
+		
+		<img src="../../assets/CN_logo.png" height="72px" alt="Coding Ninjas Logo" />
+
+        <div>
+            <p>
+                <small>
+
+                    <a href="../../index.html">Home</a>
+                    &gt; <a href="../index.html">HTML Questions</a>
+                    &gt; What is an image map?
+
+                </small>
+               
+            </p>
+        </div>
+
+
+        <content>
+            <div>
+                <h4>
+                    <h4>What is an image map?</h4>
+                </h4>
+            </div>
+            <br>
+            <br>
+            <article>
+                <strong>Answer:</strong> 
+                <p>
+                    Image map facilitates you to link many different web pages using a single image. It is represented by <map> tag. You can define shapes in images that you want to make part of an image mapping.
+                    
+                </p>
+               
+            </article>
+        </content>
+
+        <hr>
+
+        <div>
+            <h5>
+                Similar Questions
```

Comment:
@alpsz "Similar Questions" is written two times in this file i.e. some lines of code have been duplicated/repeated.
Please make changes to your answer file ASAP.